src/WhatToCook.Application/Domain/PlanOfmeals.cs

```diff
@@ -14,7 +14,7 @@ public class PlanOfMeals
 
     public List<RecipePlanOfMeals> RecipePlanOfMeals { get; private set; } = new();
 
-    public PlanOfMeals(string name, DateTime fromDate, DateTime toDate, List<RecipePerDay> recipes)
+    public PlanOfMeals(string name, DateTime fromDate, DateTime toDate, List<RecipePerDay> recipes, List<RecipePlanOfMeals> recipePlans)
```

Comment:
This change and the one starting from line 27 to 30 introduces a bug. Pls fix it